src/ring/middleware/oauth2.clj

```diff
@@ -38,10 +38,20 @@
   (= (get-in request [:session ::state])
      (get-in request [:query-params "state"])))
 
+(defn coerce-to-int
```

Comment:
This should  be a private function with no docstring, because it shouldn't be part of the public API.